include/fe/fe_base.h

```diff
@@ -591,7 +591,7 @@ class FEGenericBase : public FEAbstract
    * Compute the dual basis coefficients \p dual_coeff
    * the default qrule for element dimension and quadrature order is needed
```

Comment:
The documentation here is an atavism now, yes?

Reply:
Good catch. The docstring is outdated. Let me update it